src/point_one/rtcm/crc.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include <stdint.h>
+
+namespace point_one {
+namespace rtcm {
+
+static const uint32_t crc24q[256] = {
```

Comment:
All caps for constants `CRC24Q` (or maybe even `RTCM_CRC24Q` for clarity, even though it's in the rtcm namespace?).

Should this be `constexpr`?